ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/[externalID]/syncs/Sync/FunctionList.tsx

```diff
@@ -15,8 +25,16 @@ type Props = {
   syncedFunctions: Fn[];
 };
 
-export function FunctionList({ className, removedFunctions, syncedFunctions }: Props) {
+export function FunctionList({ removedFunctions, syncedFunctions }: Props) {
   const env = useEnvironment();
+  const [openSyncedFunctions, setSyncedFunctions] = useLocalStorage(
```

Comment:
```suggestion
  const [isSyncedFunctionsOpen, setIsSyncedFunctionsOpen] = useLocalStorage(
```

Can we remove the verb at the beginning of the variable? Reading the code later made me thing this was a function that opened synced functions, rather than a value that specified whether synced functions were already open